fairscale/nn/data_parallel/gossip/utils/helpers.py

```diff
@@ -0,0 +1,174 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+#
+
+"""
+Collection of commonly used utility functions
+"""
+
+import collections
+import logging
+import math
+import sys
+from typing import Any, Dict, List, MutableMapping, Set, Tuple
+
+import torch
+import torch.distributed as dist
+
+# Might need to install typing_extensions separately for Python 3.7 and below
+
+
+def flatten_tensors(tensors: List[torch.Tensor]) -> torch.Tensor:
```

Comment:
@vtantia may be possible to use the Flatten wrapper that we have already ?

Reply:
Could you point me to it? I'll try it out

Reply:
https://github.com/facebookresearch/fairscale/blob/master/fairscale/nn/misc/flatten_params_wrapper.py
it looks like a good fit

Reply:
(spending more time on the code, does not look like it actually, no need for a nn.Module here)